### PR TITLE
spdx license for ml-kem

### PR DESCRIPTION
**Pull request by franziskuskiefer**
cc #376

### CODE REVIEW

File: licenses.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# SPDX-FileCopyrightText: 2024 Cryspen Sarl <info@cryspen.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/bin/env bash
+
+reuse download Apache-2.0
+reuse annotate -c "Cryspen Sarl <info@cryspen.com>" -l Apache-2.0 --skip-existing --skip-unrecognised -r * .gitignore
+reuse lint
```
Comment (jschneider-bensch) [inline licenses.sh:9]: It's unfortunate that this does not allow the same configuration as `annotate`, so it shows a lot of files (e.g. all the C and F* files) as missing information.

File: libcrux-ml-kem/cg/karamel/target.h
Comment (jschneider-bensch) [inline libcrux-ml-kem/cg/karamel/target.h:1]: In this case, where copyright wasn't with Cryspen, did REUSE flag that up for you, or did you have to manually look for it?

**Review by jschneider-bensch — APPROVED** [review]
Thanks for the script as well!

**Comment by franziskuskiefer** [comment]
This got stale. I'll redo this a little cleaner.

**Pull request closed without merging**